com.unity.render-pipelines.universal/Runtime/Passes/DrawObjectsPass.cs

```diff
@@ -54,6 +56,30 @@ internal DrawObjectsPass(URPProfileId profileId, bool opaque, RenderPassEvent ev
             m_ProfilingSampler = ProfilingSampler.Get(profileId);
         }
 
+        /// <summary>
+        /// Should this pass use Depth Priming ?
+        ///
+        /// If this is true then the depth state will be overridden to only allow pixels with a depth value matching the one in the depth generated by the depth prepass while Depth writing will be turned off.
+        /// </summary>
+        public void SetUseDepthPriming(bool useDepthPriming)
+        {
+            m_UseDepthPriming = useDepthPriming;
+        }
+
+        public override void OnCameraSetup(CommandBuffer cmd, ref RenderingData renderingData)
+        {
+            if (m_UseDepthPriming)
```

Comment:
if you would expose property in renderer to access depth priming here you could do `renderingData.cameraData.renderer.useDepthPriming`.

Reply:
Fixed